ConsumerApi.Tests.Integration/API/BaseApi.cs

```diff
@@ -18,9 +19,36 @@ protected BaseApi(RestClient client)
                 (sender, cert, chain, sslPolicyErrors) => true;
     }
 
-    protected async Task<HttpResponse<T>> ExecuteRequest<T>(Method method, string endpoint, RequestConfiguration requestConfiguration)
+    protected async Task<HttpResponse<T>> Get<T>(string endpoint, RequestConfiguration requestConfiguration)
```

Comment:
I would have kept the ExecuteRequest method, made it private, and called it from the `Get` and `Post` methods.